terraform/lambda_layer.tf.json.template.py

```diff
@@ -0,0 +1,16 @@
+from azul import config
+from azul.deployment import emit_tf
+
+emit_tf({
+    "resource": [
+        {
+            "aws_lambda_layer_version": {
+                "dependencies": {
+                    "layer_name": config.qualified_resource_name("dependency_layer"),
```

Comment:
```suggestion
                    "layer_name": config.qualified_resource_name("dependencies"),
```

Reply:
AFAICT, I requested this [already](https://github.com/DataBiosphere/azul/pull/1655#discussion_r406360545). Assuming that you simply forgot to address that comment I would recommend that you first make the change and then thumb-up the comment.

Reply:
I try to be careful about this. I'll be sure I've actually made the change before adding the thumb.

Reply:
Still wrong (singular instead of plural) in latest push.